composer.json

```diff
@@ -37,6 +37,7 @@
     "require-dev": {
         "doctrine/orm": "^2.6",
         "symfony/cache": "^3.4|^4.1",
+        "symfony/translation": "^3.4|^4.1",
```

Comment:
Can be reverted now that the bugfix has been merged, right?

Reply:
I could do that if I bump the minimum Symfony versions to the latest patch release after fixes have been released. Otherwise, the prefer-lowest build would fail. Since I’m not in a hurry, I’ll just wait for the next patch releases and properly fix it then 👍